tools/clang/test/HLSLFileCheck/hlsl/diagnostics/errors/ds-error-outputpatch-size.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: %dxc -T ds_6_0 -E main %s | FileCheck %s
+// RUN: %dxc -T ds_6_0 -E main %s -spirv | FileCheck %s
+
+struct ControlPoint {
+  float position : MY_BOOL;
+};
+
+ControlPoint main(const OutputPatch<ControlPoint, 0> patch) {
+  // CHECK: error: OutputPatch element count must be greater than 0
+  return patch[0];
```

Comment:
Can you instead move this test to the `tools/clang/test/SemaHLSL` folder and make it a `-verify` test?

The clang verifier also verifies that diagnostics occur on the correct line and that there are no unexpected diagnostics emitted.

Reply:
Thanks, better like that, done!